gem/tests/test_views.py

```diff
@@ -626,72 +624,3 @@ def test_extra_params_set(self):
         self.assertEqual(view.get_extra_params(request), {
             'theme': 'springster', 'language': 'en'
         })
-
-
-class ChhaaJaaReactionQuestionsTest(TestCase, GemTestCaseMixin):
```

Comment:
Please replace this test with one that checks the same things for the new forms reaction questions

Reply:
@KaitCrawford done